cnames_active.js

```diff
@@ -3647,6 +3647,7 @@ var cnames_active = {
   "webassembly": "xtuc.github.io/webassemblyjs",
   "webbuild": "fiftys7vencode.github.io/webbuild",
   "webconnect": "nuzulul.github.io/webConnect.js",
+  "webdeepdive" "project-zyne-2025-wdd.vercel.app",
```

Comment:
```suggestion
  "webdeepdive": "cname.vercel-dns.com", // noCF
```
Two things:

1. You're missing the `:` in your entry
2. For Vercel, you'll want to use `cname.vercel-dns.com` with the Cloudflare proxy disabled via the `// noCF` comment